tools/build_updated_packages.rb

```diff
@@ -161,13 +170,12 @@ def self.agree_default_yes(message = nil)
     puts "#{name.capitalize} #{@version} has no binaries and may not need them.".lightgreen
     next pkg
   else
-    build = compatibility == 'all' ? %w[x86_64 armv7l i686] : compatibility.split
-    build.each do |arch|
+    check_builds = compatibility == 'all' ? %w[x86_64 armv7l i686] : compatibility.delete(',').split
```

Comment:
That probably doesn't work for this?
```
snort.rb:  compatibility 'aarch64,armv7l,x86_64'
```

Reply:
Was easier just to change the packages...